modules/aws/ssm_test.go

```diff
@@ -21,5 +21,21 @@ func TestParameterIsFound(t *testing.T) {
 	keyValue := GetParameter(t, awsRegion, expectedName)
 	logger.Logf(t, "Found key with name %s", expectedName)
 	assert.Equal(t, expectedValue, keyValue)
+}
+
+func TestParameterIsDeleted(t *testing.T) {
+	expectedName := fmt.Sprintf("test-name-%s", random.UniqueId())
+	// awsRegion := GetRandomRegion(t, nil, nil)
+	awsRegion := "ca-central-1"
```

Comment:
Replace with commented code

Reply:
Removed the region specific code and replaced it with `GetRandomRegion`